examples/assignment-1/src/main/java/ro/utcn/spet/example/a1/A1Application.java

```diff
@@ -0,0 +1,20 @@
+package ro.utcn.spet.example.a1;
```

Comment:
Although it is very good that you used my example, you could have renamed the top level package :) And also maybe clean up my comments from the code.

Reply:
Initially, I have tried it with "User" but it was a keyword in mysql and it kept giving me errors and as I had to change it ..."Userss" came first in my mind  :)